onnx/defs/function.cc

```diff
@@ -46,6 +47,17 @@ Common::Status FunctionBuilderRegistry::GetFunctions(
         "function_set should not be nullptr.");
   }
 
+  class FunctionBuilderRegisterer {
+   public:
+    FunctionBuilderRegisterer() {
+      RegisterOnnxFunctionBuilder();
+    }
+  };
+
+#ifndef __ONNX_DISABLE_STATIC_REGISTRATION
+  static FunctionBuilderRegisterer functionBuilder_registerer;
```

Comment:
If you want to remove the dummy class, you can write this:

`static bool functionBuilder_registerer=(RegisterOnnxFunctionBuilder(), false);`